base/bucket_gocb.go

```diff
@@ -917,13 +923,13 @@ func (bucket CouchbaseBucketGoCB) GetWithXattr(k string, xattrKey string, rv int
 			// Successfully retrieved doc and (optionally) xattr.  Copy the contents into rv, xv and return.
 			contentErr := res.Content("", rv)
 			if contentErr != nil {
-				LogTo("gocb", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, contentErr)
-				return false, contentErr, nil
+				LogTo("CRUD", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, contentErr)
+				return false, contentErr, 0
```

Comment:
This seems like a discrepancy to return a non-nil `contentErr` in conjuntion with a `0`, since the caller of the RetryLoop is doing this:

```
	err, result := RetryLoop(description, worker, bucket.spec.RetrySleeper())

	if result == nil {
		return 0, err
	}
```

And so it looks to me like it will be ignoring errors